vignettes/tutorial1.Rmd

```diff
@@ -20,8 +20,7 @@ library('fldgen')
 library('dplyr')
 library('ggplot2')
 library('reshape2')
-#dofieldplots <- require('gcammaptools')
-dofieldplots <- FALSE
+dofieldplots <- TRUE
```

Comment:
What is the point of the `dofieldplots` if else statement? Is it a left over from when we couldn't build vignettes on travis because of the gcammap tools dependency? If so do we still need it or does it make it more complicated than necessary?